File: Register people/Scripts/app/model/Person.js
Ext.define('PersonApp.model.Person', {
    extend: 'Ext.data.Model',
    fields: [{
        name: 'Id',
        type: 'int'
    }, {
        name: 'LastName',
        type: 'string',
    },{
        name: 'FirstName',
        type: 'string',
    }, {
        name: 'Patronymic',
        type: 'string',
    },{
        name:'Birthdate',
        type:'date',
        dateformat:'d.m.Y'
    }]
});

File: Register people/Models/Person.cs
﻿using System;

namespace Register_people.Models
{
    public class Person
    {
        public int? Id { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string Patronymic { get; set; }
        public DateTime Birthdate { get; set; }

    }
}